data-visualization/data-visualization.md

```diff
@@ -6,12 +6,12 @@
 
 ### Prerequisites
 
-- HTML
+- [HTML](/html/html.md)
 - CC
-- UX/UI
+- [UX/UI](https://github.com/Techtonica/curriculum/blob/master/ui-ux-design/ui-ux-design.md)
 - JavaScript
```

Comment:
@yati1998 , please link this to the javascript docs  that we have here:
https://github.com/Techtonica/curriculum/tree/master/javascript